src/material/paginator/testing/paginator-harness.ts

```diff
@@ -36,6 +36,18 @@ export abstract class _MatPaginatorHarnessBase extends ComponentHarness {
     return (await this._nextButton()).click();
   }
 
+  /** Returns whether or not the next page button is enabled. */
+  async isNextPageEnabled(): Promise<boolean> {
```

Comment:
I think that in all other places we deal with the "disabledness" of a control, not whether it is enabled.

Reply:
That was my first hunch, and for some reason I went this way.

Rallied around "Disabledness", I prefer that anyway ;D

DONE!